assets/chezmoi.io/docs/reference/templates/passhole-functions/index.md

```diff
@@ -0,0 +1,9 @@
+# passhole function
```

Comment:
The formatting of this file (and the other markdown files) does not match the style used by all other markdown files in chezmoi.